CoreScriptsRoot/Modules/PlayerlistModule.lua

```diff
@@ -1703,8 +1708,14 @@ local function onCoreGuiChanged(coreGuiType, enabled)
 	end
 end
 
-onCoreGuiChanged(Enum.CoreGuiType.PlayerList, game:GetService("StarterGui"):GetCoreGuiEnabled(Enum.CoreGuiType.PlayerList))
-game:GetService("StarterGui").CoreGuiChangedSignal:connect(onCoreGuiChanged)
+function Playerlist:TopbarEnabledChanged(enabled)
```

Comment:
Since other functions in Playerlist are using the '.' instead of ':' syntax and because Playerlist is a singleton we should continue the convention and use a '.' here for the function declaration. Please revise other scripts to match this change.